### ISSUE TITLE
제출관련 질문입니다.

### ISSUE THREAD

**Issue by gusdud1500** [snu-sf-class/pp201701#8]
![image](https://cloud.githubusercontent.com/assets/26111655/24286304/f4eb560e-10b7-11e7-9046-31d408ebdffd.png)

src.zip을 업로드하면 올리면 자동으로 파일 이름이 이상하게 변합니다.

어차피 SUCCESS가 뜨면 상관이 없겠지요?

**Comment by kim-yoonseung** [comment]
네 문제없습니다. 저 이름은 여러 번 제출할 때 이전 제출물을 보존하기 위해 임의로 붙이는 것이고, 점수와는 관련없습니다.
